spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/utils/jackson/RegistrationDeserializer.java

```diff
@@ -40,26 +40,26 @@ public Registration deserialize(JsonParser p, DeserializationContext ctxt) throw
 		JsonNode node = p.readValueAsTree();
 		Registration.Builder builder = Registration.builder();
```

Comment:
Currently the property "source" of the registration is not regarded in deserializer. Is there a reason for or it's just missing?

Reply:
iirc the source is added later in the controller and not part of the request.